documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -73,7 +73,7 @@ docker run -p 9200:9200 -d -v "$PWD/plugin_dir":/usr/share/elasticsearch/plugins
 ==== Dependencies in your Java application
 
 In addition to the usual dependencies like Hibernate ORM and Hibernate Search,
-you will need the new `hibernate-search-backend-elasticsearch` jar.
+you will need the new `hibernate-search-elasticsearch` jar.
```

Comment:
I agree we need a term for this. "Indexing service" is nice, I like it.

The change in the jar name was in prevision of the new name for the artifact.

We can commit that once Sanne has changed the name.